extensions-contrib/redis-cache/src/main/java/io/druid/client/cache/RedisCacheConfig.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Licensed to Metamarkets Group Inc. (Metamarkets) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. Metamarkets licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.druid.client.cache;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class RedisCacheConfig
+{
+    @JsonProperty
+    private String host;
+
+    @JsonProperty
+    private int port;
+
+    // seconds
+    @JsonProperty
+    private int expiration = 24 * 3600;
```

Comment:
Jedis accept seconds as `expiration`. So I can make it milliseconds, and int the code I will  modify it to seconds.  If it's OK, I'll do it.